babel.config.js

```diff
@@ -10,19 +10,19 @@ module.exports = {
     [
       '@babel/preset-env',
       {
-        targets: '> 0.5%, last 2 versions, Firefox ESR, not dead'
+        // https://browsersl.ist/#q=%3E0.3%25%2C+last+4+versions%2C+not+dead%2C+Firefox+ESR%2C+not+and_qq+13-14%2C+not+kaios+2-5%0A
+        targets: '>0.3%, last 4 versions, not dead, Firefox ESR, not and_qq 13-14, not kaios 2-5'
```

Comment:
Improves the audience coverage from [88.4%](https://browsersl.ist/#q=%3E+0.5%25%2C+last+2+versions%2C+Firefox+ESR%2C+not+dead%0A) to [89.7%](https://browsersl.ist/#q=%3E0.3%25%2C+last+4+versions%2C+not+dead%2C+Firefox+ESR%2C+not+and_qq+13-14%2C+not+kaios+2-5%0A).

Reply:
Cool. Keeping the `> 0.3%` to maintain >=80% coverage in US.


How about `> 0.3%, last 2 versions, Firefox ESR, not dead, not and_qq 13-14, not kaios 2-5`?

https://browsersl.ist/#q=%3E0.3%25%2C+last+2+versions%2C+Firefox+ESR%2C+not+dead%2C+not+and_qq+13-14%2C+not+kaios+2-5

Reply:
Yeah, I think that's a good target